spec/timezone.html

```diff
@@ -584,9 +584,9 @@ <h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-canonicalizetimezoneoffsetstring" type="abstract operation">
+    <emu-clause id="sec-temporal-NormalizeTimeZoneOffsetString" type="abstract operation">
```

Comment:
```suggestion
    <emu-clause id="sec-temporal-normalizetimezoneoffsetstring" type="abstract operation">
```

Reply:
Whoops, looks like I forgot the "match case" option when doing search-n-replace. Will fix as part of reverting the name change. 

> We might as well accept that use of "canonicalize", leaving CanonicalizeTimeZoneOffsetString as it is and using a different term to describe following time zone database Links.

Works for me. I reverted that change and just left the polyfill changes in this PR.